packages/e2e-tests/tests/file-search-01.test.ts

```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
   // Verify search results for the string "test"
   await searchSources(page, "test");
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "21 results in 1 file");
-  await verifyVisibleResultsCount(page, 25); // 21 results in 4 different files
+  await verifySourceSearchSummary(page, "12 results in 1 file");
+  await verifyVisibleResultsCount(page, 16); // 12 results in 4 different files
 
   // Verify files can be collapsed
   await toggleSearchResultsForFileName(page, false, { fileName: "jsonp%20chunk%20loading" });
   await toggleSearchResultsForFileName(page, false, { fileName: "load%20script" });
-  await verifyVisibleResultsCount(page, 22);
+  await verifyVisibleResultsCount(page, 13);
 
   // Verify files can be re-expanded
   await toggleSearchResultsForFileName(page, true, { fileName: "jsonp%20chunk%20loading" });
-  await verifyVisibleResultsCount(page, 24);
+  await verifyVisibleResultsCount(page, 15);
 
   // Now include node_modules in the search
   await toggleExcludeNodeModulesCheckbox(page, false);
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "25 results in 1 file");
-  await verifyVisibleResultsCount(page, 31); // 25 results in 6 different files
+  await verifySourceSearchSummary(page, "16 results in 1 file");
+  await verifyVisibleResultsCount(page, 22); // 16 results in 6 different files
```

Comment:
This comment is still wrong. Might as well delete it bc the assertion above has the correct summary/state.